src/SIPSorceryMedia.FFmpeg.csproj

```diff
@@ -13,17 +13,19 @@
  
   <ItemGroup>
     <PackageReference Include="DirectShowLib.Standard" Version="2.1.0" />
+    <PackageReference Include="FFmpeg.AutoGen.Bindings.DynamicallyLoaded" Version="6.0.0" />
     <PackageReference Include="SIPSorceryMedia.Abstractions" Version="1.2.1" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
-    <PackageReference Include="FFmpeg.AutoGen" Version="7.0.0" />
   </ItemGroup>
   
   <ItemGroup>
 	  <None Include="icon.png" Pack="true" PackagePath="" />
   </ItemGroup>
-
 	<PropertyGroup>
-		<TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0;net7;net8</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0;net6.0-android;net6.0-maccatalyst;net6.0-ios;net7.0;net7.0-android;net7.0-maccatalyst;net7.0-ios;net8.0;net8.0-android;net8.0-maccatalyst;net8.0-ios</TargetFrameworks>

```

Comment:
Why are all these OS specific targets needed?

Reply:
1) If we don't use FFmpeg.AutoGen.Bindings.DynamicallyLoaded a 'PlatformNotSupportedException' will be thrown at: https://github.com/Ruslan-B/FFmpeg.AutoGen/blob/15d15bdb5230fb1457883ad2b918cb94e6e1be59/FFmpeg.AutoGen/FFmpeg.cs#L17
More about that in this [issue](https://github.com/Ruslan-B/FFmpeg.AutoGen/issues/260)
2) The FFmpeg.AutoGen.Bindings.DynamicallyLoaded depends on FFmpeg.AutoGen.Abstractions so we have to use it instead of FFmpeg.AutoGen